### PR TITLE
IVI-2250 Update Java SDK for Uphold wallet and payment.

### PR DESCRIPTION
**Pull request by erik-mythical**

### CODE REVIEW

File: ivi-sdk-java/src/main/java/games/mythical/ivi/sdk/client/model/IVIUpholdCryptoCard.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@Builder
+public class IVIUpholdCryptoCard {
+    private String upholdId;
+    private String currency;
```
Comment (kwang-mythical) [inline ivi-sdk-java/src/main/java/games/mythical/ivi/sdk/client/model/IVIUpholdCryptoCard.java:16]: got one more "label" field I just added 😅

**Review by kwang-mythical — APPROVED** [review]
besides that, lgtm

**Review by keith-miller — APPROVED** [review]
lgtm

File: ivi-sdk-java/src/main/java/games/mythical/ivi/sdk/client/model/IVIUpholdCryptoCard.java (hunk 1 above)
Comment (erik-mythical) [reply ivi-sdk-java/src/main/java/games/mythical/ivi/sdk/client/model/IVIUpholdCryptoCard.java:16]: Do we want to go ahead and add that to the SDK now?

**Pull request merged**